test/js-native-api/test_finalizer/test_finalizer.c

```diff
@@ -0,0 +1,144 @@
+#include <js_native_api.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "../common.h"
+
+typedef struct {
+  int32_t finalize_count;
+  napi_ref js_func;
+} FinalizerData;
+
+static void finalizerOnlyCallback(napi_env env,
+                                  void* finalize_data,
+                                  void* finalize_hint) {
+  FinalizerData* data = (FinalizerData*)finalize_data;
+  int32_t count = ++data->finalize_count;
+
+  // It is safe to access instance data
+  NODE_API_CALL_RETURN_VOID(env, napi_get_instance_data(env, (void**)&data));
+  NODE_API_ASSERT_RETURN_VOID(
+      env, count = data->finalize_count, "Expected to the same FinalizerData");
```

Comment:
What does "Expected to the same FinalizerData" mean?

Reply:
Updated to "Expected to be the same FinalizerData".